stream-video-android-ui-compose/src/main/kotlin/io/getstream/video/android/compose/theme/base/StreamColors.kt

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2014-2024 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-video-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.video.android.compose.theme.base
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.colorResource
+import io.getstream.video.android.ui.common.R
+
+public data class StreamColors(
+    val brandPrimary: Color,
+    val brandPrimaryLt: Color,
+    val brandPrimaryDk: Color,
+    val brandSecondary: Color,
+    val brandSecondaryTransparent: Color,
+    val brandCyan: Color,
```

Comment:
It seems to have been added by mistake since we already unified theme APIs here: https://github.com/GetStream/stream-video-android/pull/1025